Library/Formula/crawl.rb

```diff
@@ -0,0 +1,37 @@
+class Crawl < Formula
+  desc "Dungeon Crawl: Stone Soup"
+  homepage "http://crawl.develz.org/"
+  version "0.17.1"
+
+  url "https://github.com/crawl/crawl.git",
+    :shallow => false,
+    :revision => "942a3d2973debefdaddbf6291e4ebc890725b920"
+
+  head "https://github.com/crawl/crawl.git",
+    :shallow => false
```

Comment:
Ah, I misunderstood that flag—I suspect it had to do with the submodule init but that's not the case.